docs/podman-rmi.1.md

```diff
@@ -43,7 +43,8 @@ podman rmi -a -f
 ## Exit Status
 **_0_** if all specified images removed
 **_1_** if one of the specified images did not exist, and no other failures
-**_125_** if command fails for a reason other then an image did not exist
+**_2_** if one of the specified images has child images or is being used by a container
+**_125_** if the command fails for a reason other than an image did not exist or is in use
```

Comment:
Should probably change and/or add to https://github.com/containers/libpod/blob/master/docs/podman.1.md#exit-status too.

Reply:
I think there should be three.  But I disagree on documenting these in podman.1.md
These exit codes are differing depending on the use case.  Exit Codes 125,126 and 127 are across multiple commands. 1,2,3 will not be the same across the commands.

Reply:
After looking at the code, it will be difficult to differentiate between. The problem is that you could end up with one that fails because it is in use, and another which fails because it has multiple layers. In which case that exit code is can not reflect both.  I think it is better to just return a single 2 value.

Reply:
@rhatdan, what I was referring to is that `rm` and `rmi` both return **2** but they are semantically different.
 
```
**_2_** if one of the specified containers is paused or running
**_2_** if one of the specified images has child images or is being used by a container
```

Imagine someone writing a wrapper for podman. Ideally, the wrapper only needs a map `error code -> error message to the user`. Having overlapping or context-dependent error codes makes this translation conditional: `error code && command -> error message to the user`.

Reply:
The original idea behind 2 was to indicate a situation where `--force` might be necessary. Although, I now realize that there's the case of having an active `exec` session that's not covered by `--force`, and I don't know how this patch set behaves in that case.